src/main/java/org/folio/rmapi/RMAPIService.java

```diff
@@ -87,13 +89,13 @@ public RMAPIService(String customerId, String apiKey, String baseURI, Vertx vert
 
       future.completeExceptionally(new RMAPIServiceException(
         String.format("%s Code = %s Message = %s Body = %s", INVALID_RMAPI_RESPONSE, response.statusCode(),
-            response.statusMessage(), body.toString()),
+          response.statusMessage(), body.toString()),
         response.statusCode(), response.statusMessage(), msgBody, query));
     }
   }
-  
+
   private String mapVendorToProvider(String msgBody) {
-    return msgBody.replaceAll(VENDOR_LOWER_STRING, PROVIDER_LOWER_STRING).replaceAll(VENDOR_UPPER_STRING, PROVIDER_UPPER_STRING);
```

Comment:
Here Sonar thinks that using regex is security vulnerability,
changed replaceAll to replace, replace method also replaces all occurences, but it uses fixed string instead of regex